### ISSUE TITLE
DistributionSchedule.WriteSet cannot be converted to List

### ISSUE THREAD

**Issue by zhaijack** [apache/distributedlog#233]
**BUG REPORT**

1. Please describe the issue you observed:

- What did you do?

run the ci job

- What did you expect to see?

ci job should be able to build successfully

- What did you see instead?

```
[INFO] -------------------------------------------------------------
[ERROR] <https://builds.apache.org/job/distributedlog-release-nightly-snapshot/ws/distributedlog-core/src/main/java/org/apache/bookkeeper/client/LedgerReader.java>:[89,69] incompatible types: org.apache.bookkeeper.client.DistributionSchedule.WriteSet cannot be converted to java.util.List<java.lang.Integer>
[ERROR] <https://builds.apache.org/job/distributedlog-release-nightly-snapshot/ws/distributedlog-core/src/main/java/org/apache/bookkeeper/client/LedgerReader.java>:[184,69] incompatible types: org.apache.bookkeeper.client.DistributionSchedule.WriteSet cannot be converted to java.util.List<java.lang.Integer>
[INFO] 2 errors
```

**Comment by zhaijack** [comment]
The problem comes from a recent change in apache/distributedlog#657. The structure of write set has been changed.

**Comment by sijie** [comment]
this is already merged by #235
